src/Show/ShowMapper.php

```diff
@@ -76,7 +80,7 @@ public function add($name, ?string $type = null, array $fieldDescriptionOptions
         } else {
             throw new \TypeError(
                 'Unknown field name in show mapper.'
-                .' Field name should be either of FieldDescriptionInterface interface or string.'
+                    .' Field name should be either of FieldDescriptionInterface interface or string.'
```

Comment:
This change looks strange, only modified indentation?

Reply:
It is strange, but it was added in https://github.com/sonata-project/SonataAdminBundle/pull/6496